config/config.default.yaml

```diff
@@ -91,23 +82,35 @@ enable:
 
 # docs in https://pypsa-eur.readthedocs.io/en/latest/configuration.html#co2-budget
 co2_budget:
-  2020: 0.720 # average emissions of 2019 to 2021 relative to 1990, CO2 excl LULUCF, EEA data, European Environment Agency. (2023a). Annual European Union greenhouse gas inventory 1990–2021 and inventory report 2023 - CRF Table. https://unfccc.int/documents/627830
-  2025: 0.648 # With additional measures (WAM) projection, CO2 excl LULUCF, European Environment Agency. (2023e). Member States’ greenhouse gas (GHG) emission projections 2023. https://www.eea.europa.eu/en/datahub/datahubitem-view/4b8d94a4-aed7-4e67-a54c-0623a50f48e8
-  2030: 0.450 # 55% reduction by 2030 (Ff55)
-  2035: 0.250
-  2040: 0.100 # 90% by 2040
-  2045: 0.050
-  2050: 0.000 # climate-neutral by 2050
+  emissions_scope: All greenhouse gases - (CO2 equivalent)
+
+  # Interpretation mode for per-period values (applies to BOTH upper and lower bounds)
+  # "fraction": Values are fractions of 1990 baseline (e.g., 0.450 = 45% of 1990)
+  # "absolute": Values are Gt CO2/year (e.g., 2.04 Gt CO2/year)
+  # Note: When enable: true, only years explicitly specified are constrained.
+  # Omitted years have no constraint for that bound type.
+  values: fraction
+
+  upper:
```

Comment:
Cases (same for lower):

1. Given pathway

```yaml
upper:
  2030: 0.5
  2040: 0.2
  2050: 0.0
```

2. No limit:

```yaml
upper: null
```

3. Total budget

```yaml
upper: 12345
```